packages/query/src/utils.ts

```diff
@@ -109,5 +109,14 @@ export const vueUnRefParams = (props: GetterProps): string => {
     .join('\n');
 };
 
+export const wrapRouteParameters = (
+  route: string,
+  prepend: string,
+  append: string,
+): string => route.replaceAll(/\${(.+?)}/g, `\${${prepend}$1${append}}`);
```

Comment:
getRouteAsArray uses a slightly different regex

Reply:
Ah, interesting. I think mine is better because it seems to be more aligned with OpenAPI specs by allowing special characters and an empty string: https://github.com/OAI/OpenAPI-Specification/issues/1969#issuecomment-509016671

But yeah, it's probably better to refactor it to avoid duplication

Reply:
I've created https://github.com/anymaniax/orval/issues/1084 to address that